docs/organizations/managing-people.md

```diff
@@ -1,17 +1,19 @@
 # Managing people
 
-Members of a Codacy organization can access the Codacy app to see the dashboards and details of the organization repositories. Depending on their permissions on the Git provider, members can also manage the organization and repository settings on Codacy.
+People signing up to the Codacy app and joining an organization become members of that organization. Members of a Codacy organization can see the dashboards and details of the organization repositories. Depending on their permissions on the Git provider, members can also manage the organization and repository settings on Codacy.
```

Comment:
This is a nice extra clarification. :star:

Reply:
I really needed that extra sentence to glue the ideas. 😅